matter_server/server/helpers/paa_certificates.py

```diff
@@ -151,11 +125,23 @@ async def fetch_git_certificates() -> int:
     """Fetch Git PAA Certificates."""
     fetch_count = 0
     LOGGER.info("Fetching the latest PAA root certificates from Git.")
-    file_list = await get_directory_contents(OWNER, REPO, PATH)
-    # Filter out extension and remove duplicates
-    git_certs = list({file.split(".")[0] for file in file_list})
+
     try:
         async with ClientSession(raise_for_status=True) as http_session:
+            # Fetch directory contents and filter out extension
+            api_url = f"https://api.github.com/repos/{OWNER}/{REPO}/contents/{PATH}"
+            async with http_session.get(api_url, timeout=20) as response:
+                if response.status == 200:
```

Comment:
We raise for status 400+. Are there other status codes that we expect that are errors?

Reply:
Not that I know of. I intentionally kept redundant error handling. Do you suggest I remove `if response.status == 200` as we already check `raise_for_status=True` ?

Reply:
Yes, possibly.